config/routes.rb

```diff
@@ -20,7 +20,7 @@
   root to: "providers#index"
   get "/organisations", to: redirect("/")
 
-  resources :notifications, path: "/notifications", controller: "notifications", only: %i[index create]
+  resources :notifications, path: "/notifications", controller: "notifications", only: %i[index update]
```

Comment:
Is there any particular reason why (possibly this has been discussed / agreed earlier and I've forgotten) why we have gone for the provider code as a param on the notifications index page?
```
http://localhost:3000/notifications?provider_code=Y50
```

rather than 
```
http://localhost:3000/notifications/Y50
```

I noticed that when I modified the provider code param, nothing would happen but I would expect either a 404 or 403

Reply:
It's only used to redirect the user back from where they came from. Did it redirect you to an existing provider?

Reply:
We could probably implement that better really and avoid the parameter being in the URL. Maybe fish it out of the referrer?

Reply:
If theres a better way then feel free to do it. I was just using it to get the basic functionality working. The idea is that we need to take them back to the previous page they were on (i.e the providers page) and display a success message